docs/how-tos/configure.md

```diff
@@ -77,25 +73,33 @@ Click on the **CONFIGURE** tab of your machine's page in the Viam app to navigat
+- Wrappers that provide additional functionality for other configured components, for example a failover motor that switches from one motor to another if the primary motor fails.
+- Software that maps to the API calls that you would use for existing hardware types. For example, a temperature sensor may provide temperature readings based on an online service instead of hardware readings. Another such type of sensor that is frequently used returns _readings_ confirming it is safe to sync data or reconfigure a machine.
+- Software that does not map well to the existing APIs, such as a switch that you'd like to be able to turn on an off. For these types of components, you use the generic component API.
+
+Follow these for each component that makes up your machine:
+
+1.  On your machine's **CONFIGURE** tab, click the **+** button next to your machine part and select **Component**.
+2.  Select the component type you need.
+3.  Select from the available models for the component.
+    You can find more information about the available models in each component's documentation.
+    For example, you can scroll through available sensor models on the [sensor page](/components/sensor/#configuration).
+
+    {{< alert title="Tip" color="tip" >}}
 
-For each component that makes up your machine:
+If a component you want to use for your project is not natively supported, you can [build your own modular resource](/how-tos/create-module/).
 
-1. Physically connect the hardware to your machine's computer.
-2. Find an appropriate model for your hardware.
-   You can find the available models on each {{< glossary_tooltip term_id="component" text="component" >}}'s documentation.
-   For example, you can scroll through available sensor models on the [sensor page](/components/sensor/#configuration).
-3. You need to [_configure_](/configure/) your machine so that `viam-server` can interact with its hardware.
-   Use the configuration builder tool in the Viam app to create a file that describes what hardware you are using and how it is connected.
-   For example, if you have a DC motor, follow the [corresponding configuration instructions](/components/motor/gpio/) to tell the software which pins it is connected to.
-4. Add a suitable model to your machine on the **CONFIGURE** page:
+    {{< /alert >}}
 
-   - Click the + icon next to your machine part in the left-hand menu and select Component.
-   - Choose any component type (example: `camera`) and model (example: `webcam`). If a component you want to use for your project is not natively supported, you can [build your own modular resource](/how-tos/create-module/).
+4.  When you add a component, it will create a panel in the configuration builder tool. Fill in any required attributes, following the documentation for the specific model.
+    You can find this documentation by following the link to the model's documentation from the available resources clicking on the respective model in each component's documentation
```

Comment:
```suggestion
    You can find this documentation by following the link to the model's documentation from the available resources clicking on the respective model in each component's documentation.
```
This sentence seems odd- is it describing two ways or the same way? could trim down second half if same thing

Reply:
```suggestion
    You can find this documentation by following the link to the model's documentation from the available resources.
```

Reply:
good catch